core/modules/widgets/browse.js

```diff
@@ -91,10 +96,27 @@ BrowseWidget.prototype.execute = function() {
 	this.nwdirectory = this.getAttribute("nwdirectory");
 };
 
+BrowseWidget.prototype.assignDomNodeClasses = function() {
+	var classes = this.getAttribute("class","").split(" ");
+	this.domNode.className = classes.join(" ");
+};
+
+BrowseWidget.prototype.assignDomNodeStyles = function() {
```

Comment:
Again, we can make this a method on the superclass in widget.js

Reply:
Refactored into widget.js @Jermolene